src/components/SubscribeToNewsLetter/SubscribeToNewsLetter.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+const SubscribeToNewsLetter = () => {
+  return (
+    <div className="flex flex-col h-screen justify-center items-center">
+      <span className="h-5 font-bold font-body text-center text-red-500 pb-10">
+        Subscribe to get our newest recipes
+      </span>
+      <div className="flex flex-row w-8/12 border border-2 rounded border-yellow-600">
+        <input
```

Comment:
this should be `required`

Reply:
the input filed should be inside a `<form />` so we can validate it later, and the form should have a `onSubmit` listener